src/pages/dar_application/DataAccessRequestApplication.jsx

```diff
@@ -143,6 +145,8 @@ const DataAccessRequestApplication = (props) => {
 
   const [applicationTabs, setApplicationTabs] = useState([]);
 
+  const [countriesOfOperation, setCountriesOfOperation] = useState([]);
```

Comment:
I would potentially put this in the `formData` so that an extra `useState` isn't required.